tests/migration_test.go

```diff
@@ -1327,13 +1327,21 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				By("Removing our migration killer pods")
 				for _, podName := range createdPods {
 					Eventually(func() error {
-						err := virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Delete(podName, &metav1.DeleteOptions{})
+						err := virtClient.CoreV1().Pods(flags.KubeVirtInstallNamespace).Delete(podName, &metav1.DeleteOptions{})
```

Comment:
does this delete work? we're creating the killer pod in tests.NamespaceTestDefault, but deleting it from flags.KubeVirtInstallNamespace.

Reply:
@davidvossel Great catch! need to fix the creation as well!